inclusive-design/idrc
# GH-454: add resources page (resolves #454, #515)

## Description

This PR adds resources page, see details on #454
This work also removes unused assts from media folder #515  

## Steps to test

1. Go to the resources page
2. See if resource cards are displaying
3. See if the resources are sorted by published year (equal published year would sort by title), and if resources don't have published year, check if they are sorted by title.

## Related issues


Resolves #454 
Resolves #515
